.env.development

```diff
@@ -24,9 +24,9 @@ SIDEKIQ_WEB_UI_PASSWORD=sidekiq
 # App store credentials (for authenticating to a local app store currently)
 # use dev app-store-auth creds.
 # TODO: OAuth requirment may be removed from local/development app store in the future
```

Comment:
As this todo has now happened, we should just remove these vars as they are not needed and all they will do is break stuff